conanfile.py

```diff
@@ -19,5 +19,9 @@ def package(self):
         self.copy(pattern="LICENSE", dst="licenses")
         self.copy(pattern="*", dst="include", src="src", keep_path=True)
 
+    def package_info(self):
+        if not self.in_local_cache:
+            self.cpp_info.includedirs = ["src"]
```

Comment:
What does these lines do exactly? I saw it's an experimental feature in Conan that is likely to change in future.